settings.gradle

```diff
@@ -7,6 +7,7 @@ include (
 		':stream-chat-android-ui-common',
 		':stream-chat-android-ui-components',
 		':stream-chat-android-ui-components-sample',
+		':stream-chat-android-ui-guides',
```

Comment:
Better naming suggestions are welcome. I understand that the "guides app" could also contain random samples, but we use the word "sample" a lot.

Reply:
Are we going to keep only ui guides here? Maybe `stream-chat-android-guides` is a better name?